bundles/org.openhab.binding.network/src/main/java/org/openhab/binding/network/internal/utils/NetworkUtils.java

```diff
@@ -174,10 +174,13 @@ public Map<String, Set<CidrAddress>> getNetworkIPsPerInterface() {
             for (Enumeration<NetworkInterface> en = NetworkInterface.getNetworkInterfaces(); en.hasMoreElements();) {
                 NetworkInterface networkInterface = en.nextElement();
                 if (networkInterface.isUp() && !networkInterface.isLoopback()) {
+                    logger.trace("Network interface: {} is included in the", networkInterface.getName());
```

Comment:
Something is missing

```suggestion
                    logger.trace("Network interface: {} is included in the search", networkInterface.getName());
```